internal/operator/clusterutilities.go

```diff
@@ -400,14 +402,16 @@ func CreatePGHAConfigMap(clientset kubernetes.Interface, cluster *crv1.Pgcluster
 	return nil
 }
 
-// sets the proper collect secret in the deployment spec if collect is enabled
-func GetCollectVolume(clientset kubernetes.Interface, cl *crv1.Pgcluster, namespace string) string {
-	if cl.Spec.UserLabels[config.LABEL_COLLECT] == "true" {
-		return "\"secret\": { \"secretName\": \"" + cl.Spec.CollectSecretName + "\" }"
+/*
+// sets the proper exporter secret in the deployment spec if exporter is enabled
+func GetExporterVolume(clientset kubernetes.Interface, cl *crv1.Pgcluster, namespace string) string {
```

Comment:
Please delete this instead of leaving it commented out.

Reply:
Got it, must have missed that in cleanup. Thanks!